tests/core/pyspec/eth2spec/test/lightclient_patch/helpers.py

```diff
@@ -0,0 +1,33 @@
+from eth2spec.test.helpers.keys import privkeys
```

Comment:
may make sense to even go ahead and move this to the `eth2spec/test/helpers` package